### DISCUSSION TITLE
LILYGO T-Deck Plus - unable to Flash / connect to flasher.meshtastic.org

### DISCUSSION THREAD

**Discussion by opcon-dev** [meshtastic/Discussions#80] [ESP32]
Hi - I received my first Meshtastic devices today :-)

My T-Deck Plus boots fine - I wanted to update the firmware to the latest stable - I restart with the trackball held down and screen is blank - I can select the device
![Screenshot from 2024-11-18 16-06-27](https://github.com/user-attachments/assets/13bf1034-a843-4a8d-8b43-000d91526920)

but it then just hangs and pulses **Disconnecting** in green in the bottom right.. 
![Screenshot from 2024-11-18 16-18-58](https://github.com/user-attachments/assets/11745ab8-ae2b-4616-9215-ff23305ea7cc)

At no point have I seen it as **Connected** I've tried the **[1200bps Reset]**, searched online and only found one other user with a similar issue, who unchecked the **Erase and Install** option - I've tried it checked and unchecked without success - any advice gratefully received!

**Comment by opcon-dev** [comment]
Hi - 

This is resolved - following the same steps it eventually recognised / connected on OSX / Mac Mini - I had previously tried on Ubuntu 22.04 LTS and on OSX - both previously said Connecting... and hung. I tried installing the two USB serial drivers on 22.04 - https://meshtastic.org/docs/getting-started/serial-drivers/esp32/ - they failed to make cleanly - but on the Mac (no additional serial drivers installed), it started to run and updated to the latest firmware. Not sure what changed.

**Reply by mariotti** [reply]
@opcon-dev you them might close the discussion. Cheers :)
